newrelic/resource_helpers_synthetics.go

```diff
@@ -74,6 +74,23 @@ func syntheticsMonitorCommonSchema() map[string]*schema.Schema {
 	}
 }
 
+func syntheticsMonitorRuntimeOptions() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"runtime_type": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			Description:  "The runtime type that the monitor will run.",
+			ValidateFunc: validation.StringInSlice([]string{"NODE_API"}, false),
+		},
+		"runtime_type_version": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			Description:  "The specific semver version of the runtime type.",
+			ValidateFunc: validation.StringInSlice([]string{"16.10"}, false),
```

Comment:
If Synthetics adds support for newer runtimes in future, these versions (comprising this validation function) would not be able to work with those newer runtimes - it might be a better idea to remove these

Reply:
Discarded